portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>Alex Vasquez</title>
+
+  <!-- Favicon -->
+  <link rel="shortcut icon" href="img/A.jpg">
+
+  <!-- load CSS Fonts -->
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" />
+  <!-- Boostrap  -->
+  <link rel="stylesheet" href="css/bootstrap.min.css" />
+
+  <!-- CSS -->
+  <link rel="stylesheet" href="css/gallery.css" />
+
+</head>
+
+<body>
+  <!-- Loader when opening page -->
+  <div id="loader-wrapper">
+    <div id="loader"></div>
+    <div class="loader-section section-left"></div>
+    <div class="loader-section section-right"></div>
+  </div>
+
+  <div class="tm-main-container">
+    <div class="tm-top-container">
+
+      <!-- Menu Button -->
+      <nav id="tmNav" class="tm-nav">
+        <a class="tm-navbar-menu" href="#">Menu</a>
+        <ul class="tm-nav-links">
+          <li class="tm-nav-item active">
+            <a href="/introduction.html" data-align="right" class="button">Home</a>
+          </li>
+
+          <li class="tm-nav-item">
+            <a href="/projects.html" data-align="right" class="button">Work and Projects</a>
+          </li>
+
+          <li class="tm-nav-item">
+            <a href="/gallery.html" data-align="middle" class="button">About Me</a>
+          </li>
+
+          <li class="tm-nav-item">
+            <a href="/contact.html" data-align="right" class="button">Contact</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Optional: Is there a way to have these common components somewhere that all the different pages can use?